app/assets/stylesheets/partials/_agenda.scss

```diff
@@ -0,0 +1,11 @@
+.agenda-grid {
+  tr {
+    &.current {
+      td,
+      &.even td,
+      &.odd td {
+        background-color: rgba(252, 152, 0, 0.5);
```

Comment:
Perhaps we should add `$orange-highlight = rgba(252, 152, 0, 0.5);` to 
https://github.com/fsek/voting/blob/master/app/assets/stylesheets/colors.scss